project/Dependencies.scala

```diff
@@ -31,7 +31,11 @@ object Dependencies {
   }
 
   object Silencer {
-    val lib    = "com.github.ghik" % "silencer-lib" % "1.6.0" % Provided cross CrossVersion.full
-    val plugin = compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.6.0" cross CrossVersion.full)
+    val lib    = ("com.github.ghik" % "silencer-lib" % "1.6.0" % Provided).cross(CrossVersion.full)
+    val plugin = compilerPlugin(("com.github.ghik" % "silencer-plugin" % "1.6.0").cross(CrossVersion.full))
+  }
+
+  object Enumeratum {
+    val enumeratum = "com.beachape" %% "enumeratum" % "1.5.15"
```

Comment:
I would prefer not to introduce a dependency unless there is a good use case for it - i don't think enumeratum adds much here and it introduces potential dependency hell etc, i would prefer to remove it